### ISSUE TITLE
Bump Web Chat to 4.15.1

### ISSUE THREAD

**Issue by compulim** [microsoft/BotFramework-Emulator#2375]
Recently, [Web Chat released 4.15.1](https://github.com/microsoft/BotFramework-WebChat/blob/main/CHANGELOG.md#4151---2022-03-04). Most of the fixes are for accessibility issues.

I hand-picked a few PRs that Emulator team may want to look at:

- https://github.com/microsoft/BotFramework-WebChat/pull/4143
   - For OAuth card, the sign-in URL must be either `http:` or `https:`
- https://github.com/microsoft/BotFramework-WebChat/pull/4108, particularly
   - A new keyboard help screen
   - Focus trap on interactive element
- https://github.com/microsoft/BotFramework-WebChat/pull/4095
   - tl;dr `Hello {1}` should now render as `Hello {1}`, instead of previously `Hello`

[enhancement]
